src/indexer.rs

```diff
@@ -221,43 +239,59 @@ impl Indexer {
             (first_block - 1, last_block, first_block - 1)
         };
 
+        // Get `current_height`
+        let current_height = if current_height == 0 {
+            if let Some(height) = self.forward_block {
+                height
+            } else {
+                0
+            }
+        } else {
+            current_height
+        };
+
         // Check, do we need fetch history data or force check from some block height
         let block = if self.force_index_from_block.is_some() || self.fetch_history {
-            if let Ok(block) = client.get_block(BlockKind::Height(num_height)).await {
+            if num_height > current_height {
+                println!(
+                    "Try to fetch block with height bigger than latest block. \
```

Comment:
It sounds like a call to action, and it is not clear how to fetch the block with a bigger height.

Reply:
fixed: https://github.com/aurora-is-near/aurora-engine-migration-tool/pull/26/commits/5de7f78f370cbf6e2a51714db19f5b80b7b1fb03